source/normalize-arguments.ts

```diff
@@ -55,42 +56,41 @@ export const preNormalizeArguments = (options: any, defaults?: any) => {
 	}
 
 	delete options.timeout;
-
 	const {retry} = options;
-	options.retry = {
-		retries: () => 0,
-		methods: new Set(),
-		statusCodes: new Set(),
-		errorCodes: new Set(),
-		maxRetryAfter: undefined
-	};
-
-	if (is.nonEmptyObject(defaults) && retry !== false) {
-		options.retry = {...defaults.retry};
-	}
 
 	if (retry !== false) {
+		options.retry = _innerDefaults.options.retry;
```

Comment:
Is this line really needed?

Reply:
Yes, in #806 default configure are missing.